packages/test/harness/src/host/jest/allocateWebDriver.js

```diff
@@ -8,9 +8,12 @@ module.exports = async function allocateWebDriver({ webDriverURL }) {
 
   preferences.setLevel(logging.Type.BROWSER, logging.Level.ALL);
 
-  const builder = new Builder()
-    .forBrowser('chrome')
-    .setChromeOptions(new ChromeOptions().addArguments('--single-process').headless().setLoggingPrefs(preferences));
+  const builder = new Builder().forBrowser('chrome').setChromeOptions(
+    new ChromeOptions()
+      .addArguments('--headless=chrome') // Should change to "--headless=new" for Chrome >= 109
```

Comment:
My setup was using 114, then 124. Are you sure we use older chrome?

Reply:
Right... our `docker-compose-wsl2.yml` is using 110.